pygfx/__init__.py

```diff
@@ -25,8 +25,8 @@
 
 del pylinalg
 
-__version__ = "0.1.18"
-version_info = tuple(map(int, __version__.split(".")))
+from ._version import __version__
+version_info = tuple(map(int, __version__.split(".")[:3]))
```

Comment:
I'm not sure what behavior you want from this `version_info` tuple. I don't have such constructs in my code.

I typically use

```
from packaging import Version
```
for all my comparisons.

Reply:
Let's do this:
```suggestion
version_info = tuple(int(v) if v.isnumeric() else v for v in __version__.split("."))
```